src/WebApp/Pages/Complaint.cshtml

```diff
@@ -6,6 +6,7 @@
 @{
     ViewData["Title"] = $"Complaint ID {Model.Item.Id.ToString()}";
 }
+<link rel="stylesheet" href="~/css/fancybox.css" />
```

Comment:
Similar to the MDE task, add a "Styles" section to the layout file for this CSS.